pytradfri/smart_task.py

```diff
@@ -46,31 +54,71 @@
 )
 
 
+class SmartTaskMixin(BaseModel):
+    """Represent common task attributes."""
+
+    state: int = Field(alias=ATTR_DEVICE_STATE)
+
+
+class StartActionResponse(BaseResponse):
+    """Represent a start action response."""
+
+    transition_time: int = Field(alias=ATTR_TRANSITION_TIME)
+    dimmer: int = Field(alias=ATTR_LIGHT_DIMMER)
+
+
+class TimeIntervalResponse(BaseModel):
+    """Represent a time interval response."""
+
+    hour_start: int = Field(alias=ATTR_SMART_TASK_TRIGGER_TIME_START_HOUR)
+    minute_start: int = Field(alias=ATTR_TIME_START_TIME_MINUTE)
+
+
+class RootStartActionResponse(SmartTaskMixin, BaseModel):
+    """Represent a smart action response."""
+
+    root_start_action: List[StartActionResponse] = Field(alias=ROOT_START_ACTION)
+
+
+class SmartTaskResponse(SmartTaskMixin, ApiResourceResponse):
```

Comment:
A sample response is available here:
https://github.com/home-assistant-libs/pytradfri/blob/master/tests/test_smart_task.py